tests/AppendStreamTest.php

```diff
@@ -67,7 +67,7 @@ public function testSeeksToPositionByReading()
         self::assertEquals('baz', $a->read(3));
```

Comment:
I'd also suggest to make the cs fixer use `assertSame` instead of `assertEquals

Reply:
Can we move this to a separate PR? because it needs a lot of changes?

Reply:
sure

Reply:
For now I changed a few easy ones.